spec/double_entry/reporting/aggregate_array_spec.rb

```diff
@@ -5,11 +5,11 @@ module Reporting
       let(:start) { nil }
       let(:finish) { nil }
       let(:range_type) { 'year' }
-      let(:function) { :sum }
+      let(:function) { 'sum' }
```

Comment:
there were repeated cases in the spec where we were casting it to a string. I thought: if we're doing that (everywhere its referenced) why not just make it a string?